lib/compilers/dotnet.ts

```diff
@@ -105,29 +142,15 @@ class DotNetCompiler extends BaseCompiler {
         }
 
         const programDir = path.dirname(inputFilename);
-        const sourceFile = path.basename(inputFilename);
 
-        const projectFilePath = path.join(programDir, `CompilerExplorer${this.lang.extensions[0]}proj`);
         const crossgen2Path = path.join(this.clrBuildDir, 'crossgen2', 'crossgen2.dll');
         const nugetConfigPath = path.join(programDir, 'nuget.config');
 
         const programOutputPath = path.join(programDir, 'bin', this.buildConfig, this.targetFramework);
         const programDllPath = path.join(programOutputPath, 'CompilerExplorer.dll');
-        const projectFileContent = `<Project Sdk="Microsoft.NET.Sdk">
-            <PropertyGroup>
-                <TargetFramework>${this.targetFramework}</TargetFramework>
-                <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-                <AssemblyName>CompilerExplorer</AssemblyName>
-                <LangVersion>${this.langVersion}</LangVersion>
-                <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
-                <Nullable>enable</Nullable>
-                <OutputType>${this.compileToBinary ? 'Exe' : 'Library'}</OutputType>
-            </PropertyGroup>
-            <ItemGroup>
-                <Compile Include="${sourceFile}" />
-            </ItemGroup>
-         </Project>
-        `;
+
+        const sdkBaseDir = path.join(path.dirname(compiler), 'sdk');
+        const sdkVersions = await fs.readdir(sdkBaseDir);
```

Comment:
unused

Reply:
ah. right, merge conflict things

Reply:
btw, how long does it take for changes to go live after the merge?

Reply:
It's a manual thing we have to do, push some buttons, test on testing environment, make fixes, deploy to live, etc. So somewhere between an hour and 2 weeks depending on the admins schedules.